### PR TITLE
Add org.deskflow.deskflow

### PR DESCRIPTION
**Pull request by sithlord48**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly. 
   Keyboard and mouse sharing , Software KVM
- [X] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

closes https://github.com/deskflow/deskflow/issues/7593

### CODE REVIEW

File: org.deskflow.deskflow.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    sources:
+      - type: git
+        url: https://github.com/flatpak/libportal.git
+        branch: 0.8.1
+  - name: puixml
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/zeux/pugixml
+        branch: v1.14
```
[the remark this answers is not in the document]
Comment (sithlord48) [reply org.deskflow.deskflow.yml:79]: should that be a tag or just nothing ?

File: org.deskflow.deskflow.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+app-id: org.deskflow.deskflow
+runtime: org.kde.Platform
+runtime-version: "6.7"
```
Comment (hfiguiere) [inline org.deskflow.deskflow.yml:3]: 6.8 is latest

File: org.deskflow.deskflow.yml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+  - --talk-name=org.a11y.*
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: libnotify
```
Comment (hfiguiere) [inline org.deskflow.deskflow.yml:16]: this is in the runtime

Comment (hfiguiere) [reply org.deskflow.deskflow.yml:16]:
```
[📦 org.kde.Sdk ~]$ pkg-config --modversion libnotify
0.8.3
```

File: org.deskflow.deskflow.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hfiguiere) [reply org.deskflow.deskflow.yml:79]: commit or tag+commit. Anything is blocked by the linter.

Not having a branch is as bad.

File: org.deskflow.deskflow.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    cleanup:
+      - '*'
+  - name: deskflow
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=Release"
+    sources:
+      - type: git
+        url: https://github.com/deskflow/deskflow.git
+        branch: master
```
Comment (hfiguiere) [inline org.deskflow.deskflow.yml:106]: still. `branch` is NOT allowed.

File: org.deskflow.deskflow.yml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  - name: libportal
+    buildsystem: meson
+    config-opts:
+      - -Ddocs=false
+      - -Dtests=false
+      - -Dportal-tests=false
+      - -Dvapi=false
+      - -Dbackend-qt5=auto
+      - -Dbackend-qt6=auto
+      - -Dbackend-gtk4=auto
```
Comment (hfiguiere) [inline org.deskflow.deskflow.yml:61]:
```suggestion
      - -Dbackend-qt6=auto
```

File: org.deskflow.deskflow.yml [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+command: deskflow
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+  - --talk-name=org.a11y.*
```
Comment (bbhtt) [inline org.deskflow.deskflow.yml:12]: why is this needed?

File: org.deskflow.deskflow.yml (hunk 4 above)
Comment (hfiguiere) [reply org.deskflow.deskflow.yml:106]: then we can't build it.

Comment (hfiguiere) [reply org.deskflow.deskflow.yml:106]: not sure why you are pushing back here.

File: org.deskflow.deskflow.yml (hunk 6 above)
Comment (sithlord48) [reply org.deskflow.deskflow.yml:12]: its not ill removed

File: org.deskflow.deskflow.yml (hunk 5 above)
Comment (sithlord48) [reply org.deskflow.deskflow.yml:61]: only qt6 ? done

Comment (hfiguiere) [reply org.deskflow.deskflow.yml:61]: where are the others ?

*hint* not in this runtime. Only gtk3. 

But that's about what the app uses anyway.

File: org.deskflow.deskflow.yml (hunk 4 above)
Comment (sithlord48) [reply org.deskflow.deskflow.yml]: 1.17.2 is released tag updated

**Comment by bbhtt** [comment]
bot, build org.deskflow.deskflow
